### PR TITLE
Refactored Appointments page

### PR DESCRIPTION
**Pull request by Priya-81199**
Created appointments_view and appointments_viewmodel inside the Appointments folder

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: scp/lib/services/firebase_analytics_service.dart [hunk 1]
```diff
@@ -5,4 +5,4 @@ class AnalyticsService{
   final FirebaseAnalytics _analytics = FirebaseAnalytics();
   
   FirebaseAnalyticsObserver getAnalyticsObserver() => FirebaseAnalyticsObserver(analytics: _analytics);
-}
+}
```
Comment (Thesmader) [inline scp/lib/services/firebase_analytics_service.dart:8]:
```suggestion
}

```

File: scp/lib/Appointments/appointments_viewmodel.dart [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    _onCounselChangedSubscription.cancel();
+    _onPsychChangedSubscription.cancel();
+  }
+
+}
```
Comment (SriramPatibanda) [inline scp/lib/Appointments/appointments_viewmodel.dart:68]: Add a blank line at the end of every file.
```suggestion
}

```

File: scp/lib/Appointments/appointments_viewmodel.dart [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  @override
+  void initState() {
+    /*ScpDatabase().init();*/
+    getDate();
+    //isBookingAnonymously = false;
+    scpDatabase = ScpDatabase();
+    //super.initState();
+    notifyListeners();
+  }
+  getDate() async {
```
Comment (SriramPatibanda) [inline scp/lib/Appointments/appointments_viewmodel.dart:30]:
```suggestion
 void getDate() async {
```

File: scp/lib/Appointments/appointments_viewmodel.dart [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    RemoteConfig remoteConfig = await RemoteConfig.instance;
+    // Enable developer mode to relax fetch throttling
+    remoteConfig.setConfigSettings(RemoteConfigSettings(debugMode: true));
+    await remoteConfig.fetch(expiration: const Duration(seconds: 0));
+    await remoteConfig.activateFetched();
+    counselDay = remoteConfig.getString('counsel_day');
+    counselorName = remoteConfig.getString('counselor_name');
+    psychName = remoteConfig.getString('psych_name');
+    psychDay = remoteConfig.getString('psych_day');
+    print(counselDay + 'hola');
```
Comment (SriramPatibanda) [inline scp/lib/Appointments/appointments_viewmodel.dart:48]: try not to print anything to the console.
```suggestion

```

**Review by SriramPatibanda — CHANGES_REQUESTED** [review]
Have suggested very minor changes, please do those and commit, apart from that the code looks good. Great job 👍🏻

**Review by SriramPatibanda — APPROVED** [review]

**Comment by DesignrKnight** [comment]
image
Can you resolve these?

**Review by Thesmader — APPROVED** [review]
LGTM

**Pull request merged**